ipm_library/ipm_library/ipm.py

```diff
@@ -71,8 +71,13 @@ def project_point(
         :param plane: Plane in which the projection should happen
         :param point: Point that should be projected
         :param output_frame: TF2 frame in which the output should be provided
+        :raise: InvalidPlaneException if the plane is invalid
+        :raise: NoIntersectionError if the point is not on the plane
         :returns: The point projected onto the given plane in the output frame
         """
+        if plane.plane.coef[0] == 0 and plane.plane.coef[1] == 0 and plane.plane.coef[2] == 0:
```

Comment:
This line is a bit long. Maybe we can do something like this:

```python
if not any(plane.plane.coef[:3]):
```

Reply:
Also as this is just wrapping `project_points` we can skip this here and let the exception propergate form there instead of checking twice.

Reply:
I agree with both of your suggestions, thanks!

Reply:
I've made the changes.